### PR TITLE
made first example of solution

### PR DESCRIPTION
**Pull request by Sashk52**

### CODE REVIEW

File: src/main/java/com/dev/movietheatre/dao/impl/MovieDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert Movie entity", e);
```
Comment (mfedenko09) [inline src/main/java/com/dev/movietheatre/dao/impl/MovieDaoImpl.java]:
```suggestion
            throw new RuntimeException("Can't insert Movie entity" + movie, e);
```
and add own exception

File: src/main/java/com/dev/movietheatre/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    @Override
+    public List<Movie> getAll() {
+        try {
+            Session session = HibernateUtil.getSessionFactory().openSession();
+            Query<Movie> getAllMoviesQuery = session.createQuery("from Movie", Movie.class);
+            return getAllMoviesQuery.getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get list of all movies ", e);
```
Comment (mfedenko09) [inline src/main/java/com/dev/movietheatre/dao/impl/MovieDaoImpl.java]:
```suggestion
            throw new RuntimeException("Can't get list of all movies ", e);
```
add own exception

File: src/main/java/com/dev/movietheatre/util/HibernateUtil.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new RuntimeException("Error creating session Factory", e);
```
Comment (mfedenko09) [inline src/main/java/com/dev/movietheatre/util/HibernateUtil.java]:
```suggestion
            throw new RuntimeException("Error creating session Factory", e);
```
same here

File: src/main/resources/hibernate.cfg.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        <property name="connection.url">jdbc:mysql://localhost/movie_theatre?serverTimezone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">1234</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="com.dev.movietheatre.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (mfedenko09) [inline src/main/resources/hibernate.cfg.xml]: add empty line

**Review by mfedenko09 — CHANGES_REQUESTED** [review]

File: src/main/resources/hibernate.cfg.xml (hunk 4 above)
Comment (Sashk52) [reply src/main/resources/hibernate.cfg.xml]: done

File: src/main/java/com/dev/movietheatre/util/HibernateUtil.java (hunk 3 above)
Comment (Sashk52) [reply src/main/java/com/dev/movietheatre/util/HibernateUtil.java]: done

File: src/main/java/com/dev/movietheatre/dao/impl/MovieDaoImpl.java (hunk 2 above)
Comment (Sashk52) [reply src/main/java/com/dev/movietheatre/dao/impl/MovieDaoImpl.java]: done

File: src/main/java/com/dev/movietheatre/dao/impl/MovieDaoImpl.java (hunk 1 above)
Comment (Sashk52) [reply src/main/java/com/dev/movietheatre/dao/impl/MovieDaoImpl.java]: done

**Review by mfedenko09 — APPROVED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**